app/common/lib/historyUtil.js

```diff
@@ -15,20 +15,32 @@ const sortTimeDescending = (left, right) => {
 }
 
 const getHistory = (sites) => {
-  sites = makeImmutable(sites) ? makeImmutable(sites).toList() : new Immutable.List()
+  if (sites == null) {
+    return Immutable.List()
+  }
+
+  sites = makeImmutable(sites) ? makeImmutable(sites).toList() : Immutable.List()
```

Comment:
I know this logic is the same as before, but I think it's wrong.  Should it be:
`sites = sites ? makeImmutable(sites).toList() : Immutable.List()`

Reply:
(makeImmutable won't return falsey values basically)

Reply:
Since the above is checking == null (which should also return true for undefined, right?), I think we can simply make it
`sites = makeImmutable(sites).toList()`

Reply:
fixed